tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py34,py35,py36,pep8,pypy
+envlist = py27,py34,py35,py36,py37,py38,pep8,pypy
```

Comment:
you might want to remove py34 and py35 we don't support.

Reply:
Sure, updated.